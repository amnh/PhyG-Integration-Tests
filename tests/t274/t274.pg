--build(graph:softwired), inputting fastc files along with tcm
set(seed:1634561640)
set(graphtype:softwired)
read(rename:"../data/woman_rename.txt")
read(fastc:"../data/woman.fastc", tcm:"../data/woman-11.tcm")
read(fastc:"../data/woman_cox1_spaced.fastc")
read(include:"../data/Include_woman16.txt")
build(distance, dwag, block, cun, displaytrees:10, atrandom, graph)
report("t274.csv", data, overwrite)
report("t274.tre", newick, displaytrees, overwrite)
report("t274.dot", dotpdf, overwrite)