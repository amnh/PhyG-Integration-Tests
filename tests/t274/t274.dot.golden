digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=2.0];
    16 -> 29 [label=5.0];
    17 -> 4 [label=29.5];
    17 -> 11 [label=20.5];
    18 -> 1 [label=42.0];
    18 -> 9 [label=32.0];
    19 -> 13 [label=29.0];
    19 -> 18 [label=30.0];
    20 -> 21 [label=25.0];
    20 -> 30 [label=21.0];
    21 -> 22 [label=11.0];
    21 -> 23 [label=16.5];
    22 -> 12 [label=32.5];
    22 -> 24 [label=18.5];
    23 -> 25 [label=18.0];
    23 -> 27 [label=19.5];
    24 -> 7 [label=25.5];
    24 -> 8 [label=58.5];
    25 -> 17 [label=24.5];
    25 -> 28 [label=23.0];
    26 -> 6 [label=33.5];
    26 -> 10 [label=47.5];
    27 -> 3 [label=43.5];
    27 -> 14 [label=38.5];
    28 -> 15 [label=33.0];
    28 -> 26 [label=15.5];
    29 -> 5 [label=41.5];
    29 -> 20 [label=17.0];
    30 -> 2 [label=28.5];
    30 -> 19 [label=23.5];
}
//774.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=2.0];
    16 -> 29 [label=5.0];
    17 -> 4 [label=29.5];
    17 -> 11 [label=20.5];
    18 -> 1 [label=42.0];
    18 -> 9 [label=32.0];
    19 -> 13 [label=29.0];
    19 -> 18 [label=30.0];
    20 -> 21 [label=25.0];
    20 -> 30 [label=21.0];
    21 -> 22 [label=11.0];
    21 -> 23 [label=16.5];
    22 -> 12 [label=32.5];
    22 -> 24 [label=18.5];
    23 -> 25 [label=18.0];
    23 -> 27 [label=19.5];
    24 -> 7 [label=25.5];
    24 -> 8 [label=58.5];
    25 -> 17 [label=24.5];
    25 -> 28 [label=23.0];
    26 -> 6 [label=33.5];
    26 -> 10 [label=47.5];
    27 -> 3 [label=43.5];
    27 -> 14 [label=38.5];
    28 -> 15 [label=33.0];
    28 -> 26 [label=15.5];
    29 -> 5 [label=41.5];
    29 -> 20 [label=17.0];
    30 -> 2 [label=28.5];
    30 -> 19 [label=23.5];
}
//774.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=2.0];
    16 -> 29 [label=5.0];
    17 -> 4 [label=29.5];
    17 -> 11 [label=20.5];
    18 -> 1 [label=42.0];
    18 -> 9 [label=32.0];
    19 -> 13 [label=29.0];
    19 -> 18 [label=30.0];
    20 -> 21 [label=25.0];
    20 -> 30 [label=21.0];
    21 -> 22 [label=11.0];
    21 -> 23 [label=16.5];
    22 -> 12 [label=32.5];
    22 -> 24 [label=18.5];
    23 -> 25 [label=18.0];
    23 -> 27 [label=19.5];
    24 -> 7 [label=25.5];
    24 -> 8 [label=58.5];
    25 -> 17 [label=24.5];
    25 -> 28 [label=23.0];
    26 -> 6 [label=33.5];
    26 -> 10 [label=47.5];
    27 -> 3 [label=43.5];
    27 -> 14 [label=38.5];
    28 -> 15 [label=33.0];
    28 -> 26 [label=15.5];
    29 -> 5 [label=41.5];
    29 -> 20 [label=17.0];
    30 -> 2 [label=28.5];
    30 -> 19 [label=23.5];
}
//774.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=2.0];
    16 -> 29 [label=5.0];
    17 -> 4 [label=29.5];
    17 -> 11 [label=20.5];
    18 -> 1 [label=42.0];
    18 -> 9 [label=32.0];
    19 -> 13 [label=29.0];
    19 -> 18 [label=30.0];
    20 -> 21 [label=25.0];
    20 -> 30 [label=21.0];
    21 -> 22 [label=11.0];
    21 -> 23 [label=16.5];
    22 -> 12 [label=32.5];
    22 -> 24 [label=18.5];
    23 -> 25 [label=18.0];
    23 -> 27 [label=19.5];
    24 -> 7 [label=25.5];
    24 -> 8 [label=58.5];
    25 -> 17 [label=24.5];
    25 -> 28 [label=23.0];
    26 -> 6 [label=33.5];
    26 -> 10 [label=47.5];
    27 -> 3 [label=43.5];
    27 -> 14 [label=38.5];
    28 -> 15 [label=33.0];
    28 -> 26 [label=15.5];
    29 -> 5 [label=41.5];
    29 -> 20 [label=17.0];
    30 -> 2 [label=28.5];
    30 -> 19 [label=23.5];
}
//774.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=2.0];
    16 -> 29 [label=5.0];
    17 -> 10 [label=50.0];
    17 -> 20 [label=22.5];
    18 -> 4 [label=29.0];
    18 -> 11 [label=21.0];
    19 -> 1 [label=41.0];
    19 -> 9 [label=33.0];
    20 -> 13 [label=31.5];
    20 -> 19 [label=22.0];
    21 -> 22 [label=12.0];
    21 -> 30 [label=20.5];
    22 -> 23 [label=12.0];
    22 -> 24 [label=15.5];
    23 -> 12 [label=33.0];
    23 -> 25 [label=20.0];
    24 -> 26 [label=17.0];
    24 -> 27 [label=16.0];
    25 -> 7 [label=26.5];
    25 -> 8 [label=57.5];
    26 -> 18 [label=25.5];
    26 -> 28 [label=18.0];
    27 -> 3 [label=53.5];
    27 -> 14 [label=32.5];
    28 -> 6 [label=27.0];
    28 -> 15 [label=44.0];
    29 -> 5 [label=38.5];
    29 -> 21 [label=18.5];
    30 -> 2 [label=27.0];
    30 -> 17 [label=23.0];
}
//773.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=2.0];
    16 -> 29 [label=5.0];
    17 -> 10 [label=50.0];
    17 -> 20 [label=22.5];
    18 -> 4 [label=29.0];
    18 -> 11 [label=21.0];
    19 -> 1 [label=41.0];
    19 -> 9 [label=33.0];
    20 -> 13 [label=31.5];
    20 -> 19 [label=22.0];
    21 -> 22 [label=12.0];
    21 -> 30 [label=20.5];
    22 -> 23 [label=12.0];
    22 -> 24 [label=15.5];
    23 -> 12 [label=33.0];
    23 -> 25 [label=20.0];
    24 -> 26 [label=17.0];
    24 -> 27 [label=16.0];
    25 -> 7 [label=26.5];
    25 -> 8 [label=57.5];
    26 -> 18 [label=25.5];
    26 -> 28 [label=18.0];
    27 -> 3 [label=53.5];
    27 -> 14 [label=32.5];
    28 -> 6 [label=27.0];
    28 -> 15 [label=44.0];
    29 -> 5 [label=38.5];
    29 -> 21 [label=18.5];
    30 -> 2 [label=27.0];
    30 -> 17 [label=23.0];
}
//773.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=1.5];
    16 -> 29 [label=4.5];
    17 -> 4 [label=27.0];
    17 -> 11 [label=23.0];
    18 -> 1 [label=50.5];
    18 -> 13 [label=23.5];
    19 -> 20 [label=21.0];
    19 -> 30 [label=19.0];
    20 -> 21 [label=13.0];
    20 -> 22 [label=16.5];
    21 -> 12 [label=33.5];
    21 -> 23 [label=19.5];
    22 -> 24 [label=20.0];
    22 -> 27 [label=16.5];
    23 -> 7 [label=25.0];
    23 -> 8 [label=59.0];
    24 -> 25 [label=19.0];
    24 -> 28 [label=19.5];
    25 -> 9 [label=25.5];
    25 -> 17 [label=25.5];
    26 -> 6 [label=33.0];
    26 -> 10 [label=48.0];
    27 -> 3 [label=42.5];
    27 -> 14 [label=39.5];
    28 -> 15 [label=31.0];
    28 -> 26 [label=16.5];
    29 -> 5 [label=42.0];
    29 -> 19 [label=21.0];
    30 -> 2 [label=29.0];
    30 -> 18 [label=24.0];
}
//769.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=1.5];
    16 -> 29 [label=29.0];
    17 -> 10 [label=49.0];
    17 -> 19 [label=24.0];
    18 -> 4 [label=26.0];
    18 -> 11 [label=24.0];
    19 -> 1 [label=43.5];
    19 -> 13 [label=29.5];
    20 -> 21 [label=8.5];
    20 -> 30 [label=17.0];
    21 -> 22 [label=14.0];
    21 -> 23 [label=15.5];
    22 -> 12 [label=34.0];
    22 -> 24 [label=20.0];
    23 -> 25 [label=22.0];
    23 -> 27 [label=14.0];
    24 -> 7 [label=26.5];
    24 -> 8 [label=57.5];
    25 -> 26 [label=17.5];
    25 -> 28 [label=13.0];
    26 -> 9 [label=25.5];
    26 -> 18 [label=24.5];
    27 -> 3 [label=43.0];
    27 -> 14 [label=39.0];
    28 -> 6 [label=30.0];
    28 -> 15 [label=41.0];
    29 -> 5 [label=50.0];
    29 -> 20 [label=14.5];
    30 -> 2 [label=30.0];
    30 -> 17 [label=21.5];
}
//768.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=1.5];
    16 -> 29 [label=29.0];
    17 -> 10 [label=49.0];
    17 -> 19 [label=24.0];
    18 -> 4 [label=26.0];
    18 -> 11 [label=24.0];
    19 -> 1 [label=43.5];
    19 -> 13 [label=29.5];
    20 -> 21 [label=8.5];
    20 -> 30 [label=17.0];
    21 -> 22 [label=14.0];
    21 -> 23 [label=15.5];
    22 -> 12 [label=34.0];
    22 -> 24 [label=20.0];
    23 -> 25 [label=22.0];
    23 -> 27 [label=14.0];
    24 -> 7 [label=26.5];
    24 -> 8 [label=57.5];
    25 -> 26 [label=17.5];
    25 -> 28 [label=13.0];
    26 -> 9 [label=25.5];
    26 -> 18 [label=24.5];
    27 -> 3 [label=43.0];
    27 -> 14 [label=39.0];
    28 -> 6 [label=30.0];
    28 -> 15 [label=41.0];
    29 -> 5 [label=50.0];
    29 -> 20 [label=14.5];
    30 -> 2 [label=30.0];
    30 -> 17 [label=21.5];
}
//768.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=1.5];
    16 -> 29 [label=29.0];
    17 -> 10 [label=49.0];
    17 -> 19 [label=24.0];
    18 -> 4 [label=26.0];
    18 -> 11 [label=24.0];
    19 -> 1 [label=43.5];
    19 -> 13 [label=29.5];
    20 -> 21 [label=8.5];
    20 -> 30 [label=17.0];
    21 -> 22 [label=14.0];
    21 -> 23 [label=15.5];
    22 -> 12 [label=34.0];
    22 -> 24 [label=20.0];
    23 -> 25 [label=22.0];
    23 -> 27 [label=14.0];
    24 -> 7 [label=26.5];
    24 -> 8 [label=57.5];
    25 -> 26 [label=17.5];
    25 -> 28 [label=13.0];
    26 -> 9 [label=25.5];
    26 -> 18 [label=24.5];
    27 -> 3 [label=43.0];
    27 -> 14 [label=39.0];
    28 -> 6 [label=30.0];
    28 -> 15 [label=41.0];
    29 -> 5 [label=50.0];
    29 -> 20 [label=14.5];
    30 -> 2 [label=30.0];
    30 -> 17 [label=21.5];
}
//768.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=AlbanianTosk];
    1 [label=AncientIonicGreek];
    2 [label=ArmenianWestern];
    3 [label=Assamese];
    4 [label=Cornish];
    5 [label=Danish];
    6 [label=EasternFrisianSaterland];
    7 [label=Faroese];
    8 [label=Icelandic];
    9 [label=Irish];
    10 [label=KolschColognian];
    11 [label=Manx];
    12 [label=OldPrussian];
    13 [label=SanskritVedic];
    14 [label=Scots];
    15 [label=Yiddish];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    16 -> 0 [label=1.5];
    16 -> 30 [label=29.0];
    17 -> 19 [label=24.0];
    17 -> 31 [label=10.0];
    18 -> 4 [label=26.0];
    18 -> 11 [label=24.0];
    19 -> 1 [label=43.0];
    19 -> 13 [label=30.0];
    20 -> 21 [label=8.0];
    20 -> 32 [label=18.0];
    21 -> 22 [label=14.0];
    21 -> 23 [label=15.0];
    22 -> 12 [label=34.0];
    22 -> 24 [label=20.0];
    23 -> 25 [label=22.0];
    23 -> 28 [label=14.0];
    24 -> 7 [label=26.5];
    24 -> 8 [label=57.5];
    25 -> 26 [label=17.5];
    25 -> 29 [label=13.0];
    26 -> 9 [label=25.5];
    26 -> 18 [label=24.5];
    27 -> 6 [label=30.0];
    27 -> 31 [label=1.5];
    28 -> 3 [label=43.0];
    28 -> 14 [label=39.0];
    29 -> 15 [label=41.0];
    29 -> 27 [label=5.0];
    30 -> 5 [label=50.0];
    30 -> 20 [label=15.0];
    31 -> 10 [label=46.0];
    32 -> 2 [label=30.5];
    32 -> 17 [label=20.5];
}
//767.0
